backend/db/migrations/20200208203843-create-scholarship.js

```diff
@@ -0,0 +1,63 @@
+"use strict";
+
+module.exports = {
+  up: (queryInterface, Sequelize) => {
+    return queryInterface.createTable("scholarships", {
+      id: {
+        allowNull: false,
+        autoIncrement: true,
+        primaryKey: true,
+        type: Sequelize.INTEGER
+      },
+      name: {
+        type: Sequelize.STRING,
```

Comment:
We might want to make a habit of giving default values to non nullable columns. It doesn't break anything directly if theres no default, but I recently ran into a weird problem with entry related to non-nullable fields without defaults